### ISSUE TITLE
Add to Scoop

### ISSUE THREAD

**Issue by sbstn87** [vemonet/EmojiMart#1]
Hi,

could you perhaps add your app to Scoop?

https://scoop.sh/

**Comment by vemonet** [comment]
Hi @sbstn87 , thanks for the recommendation, I currently did not try it on windows (even if the .exe compiles properly), so I am not sure in which state it is on Windows (I'll try it soon when I have the time)

Did you try it on windows? Is it working as expected?

Scoop packaging seems quite straightforward: https://scoop.sh/ they mention a `.zip` as download file URL though, I hope we can do it directly with the `.exe`. I'll check it out more in details once we know if it runs well on windows :)

**Comment by sbstn87** [comment]
Yes, it works correctly for me on Windows 10. At least I suppose it does - whenever I click somewhere else than the app window it closes automatically.
